appium/options/flutter/flutter_element_wait_timeout_option.py

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import Optional
+from appium.options.common.supports_capabilities import SupportsCapabilities
+
+
+FLUTTER_ELEMENT_WAIT_TIMEOUT= 'flutterElementWaitTimeout'
+
+
+class FlutterElementWaitTimeOutOption(SupportsCapabilities):
+    
+    @property
+    def flutter_element_wait_timeout(self) -> Optional[int]:
+        return self.get_capability(FLUTTER_ELEMENT_WAIT_TIMEOUT)
+
+    @flutter_element_wait_timeout.setter
+    def flutter_element_wait_timeout(self, time_in_millis: int) -> None:
```

Comment:
Please change the variable type to Union[int, timedelta], like it is done for other similar time-related capabilities. See, for example, https://github.com/appium/python-client/blob/master/appium/options/android/uiautomator2/uiautomator2_server_install_timeout_option.py